chapters/hpmor-chapter-012.tex

```diff
@@ -1,10 +1,10 @@
 \chapter{Impulse Control}
 
 \begin{chapterOpeningAuthorNote}
-ph'nglui mglw'nafh J.~K.~Rowling wgah'nagl fhtagn
+ph‘nglui mglw’nafh J.~K.~Rowling wgah'nagl fhtagn
```

Comment:
Quote "wgah'nagl" needs curling.

Reply:
fixed in https://github.com/rrthomas/hpmor/pull/85/